README.md

```diff
@@ -82,7 +82,24 @@ A step by step series that tell you how to get Project Groot up and running
 4. navigate to the sync repo in vm
 
 ```
-   cd /vagrant/GIVIT/
+   cd /vagrant/givitsite/
+```
+
+5. Creating an admin user
+
+First we’ll need to create a user who can login to the admin site. Run the following command:
+
+```
+   python3 manage.py createsuperuser
```

Comment:
I recommend doing this via a migration - see here for an idea how to do that: https://stackoverflow.com/a/53555252/8243700

Reply:
i search and read about it a little and it sound like a better approach, the problem is that i need to wait that nadav will merge his PR to be able to test and make sure that its work. 

i suggest that we live it like this for now and after nadav will merge i will do the updates with this aproach, its sound ok to you?

Reply:
ok, but mention that in the commit message, and create a GitHub ticket to track the future action item (and mention it here as well)

Reply:
there is more names for ticket? i see also that you mention it in SOW but not explain how to do it. and i search it in google and can't find what it is.. 
can you send me some leads to start from? thanks :)

Reply:
I GitHub they are called "issues" you create them from the tab above. Its between the "Code" tab and the "Pull requests" tab.

You should generally be creating issues for every piece of work that you do, its one of the key mechanisms by which your team can track who does what when.

Reply:
done.

Reply:
Pleas link the issue here